jcache/commons/src/main/java/org/infinispan/jcache/annotation/AbstractInjectedCacheResolver.java

```diff
@@ -0,0 +1,10 @@
+package org.infinispan.jcache.annotation;
+
+import java.lang.annotation.Annotation;
+
+import javax.cache.Cache;
+import javax.cache.annotation.CacheInvocationContext;
+import javax.cache.annotation.CacheResolver;
+
+public abstract class AbstractInjectedCacheResolver implements CacheResolver {
```

Comment:
It's just a placeholder (no remote implementation yet) but there's a subclass for the embedded cache. Without the abstract class I would need to move org.infinispan.jcache.annotations out of infinispan-jcache-commons.